addons/framework/buildccp/initmed.sqf

```diff
@@ -46,51 +49,14 @@ if (_playerClass in _medics) then {
 		[player] call tnk_add_med_mo;
 	};
 	//Alpha medics
-	if (_playerClass in ["BLU_ASLM","OPF_ASLM","IND_ASLM"]) then {
+	if (_playerClass in _medics) then {
```

Comment:
Looks good. Fix the indentation/whitespace here and we can merge

Reply:
Did some testing, and after running into issues convened with patman and found an issue with classes being private. 

Fixed in newest commit. 

Everything is locally tested, working, and throwing no errors. 

Should hopefully be good to go